frontend/src/components/pages/rp-connect/onboarding/add-secrets-card.tsx

```diff
@@ -85,14 +106,48 @@ export const AddSecretsCard = ({
     },
     [editorInstance]
   );
+
+  const handleSecretsCreated = useCallback(() => {
+    refetchSecrets();
+    setIsSecretsDialogOpen(false);
+  }, [refetchSecrets]);
```

Comment:
Let's invalidate the relevant connect query cache keys instead, no need to have refetch triggered manually, WDYT?